### ISSUE TITLE
OCAD graphic objects are imported as unknown symbol

### ISSUE THREAD

**Issue by lpechacek** [OpenOrienteering/mapper#959]
I've just received an OCAD 10 map where some of the fences are drawn as so called Graphic Objects. They get imported into OOM as lines with undefined symbol.

Looking into [OCAD 10 format description](http://www.ocad.ch/docs/OCAD10Format.txt), I seems to me that graphic objects are merely lines with defined color and width. It would be great is OOM either issued warning when encountering these objects or synthesized symbol definitions for them.

### Configuration
Mapper Version: v0.7.0
Operating System: openSUSE Tumbleweed

**Comment by dg0yt** [comment]
This will need test files.

In order to not pollute the defined symbol set, an implicit symbol definition would need to embedded into the individual object. This should be aligned with other changes for dealing with graphic primitives, such as copy-and-paste between map and symbol editor.

**Comment by lpechacek** [comment]
Shared a sample file via e-mail.

**Comment by krticka** [comment]
This issue is also connected wit graphic area objects. For example I received Ocad file with map and separate file with design, where some texts were converted to graphics to avoid unwanted font substitution. When you open the file with design in Mapper, only outlines are visible. When you open the map file, and design file open as template, all graphic objects are missing. 
Design file in OCAD:
![Ocad](https://user-images.githubusercontent.com/15969434/56083521-66531580-5e26-11e9-9409-4fbf768a88cc.jpg)

Design file in Mapper:
![Mapper](https://user-images.githubusercontent.com/15969434/56083526-710daa80-5e26-11e9-8b5f-ba99fe9b0393.jpg)

**Comment by lpechacek** [comment]
> I tried to create a sample .ocd12 file for testing your commit but ended up with getting "layout vector objects" (-4). 

And yes, I'm slow at getting the whole message. Ad creation graphic objects for testing, I think you should draw a few complex lines (earth banks, streets), some point objects, and some text and [convert them to graphic objects](https://www.ocad.com/wiki/ocad/en/index.php?title=Convert_into_Graphic_Object). That should create objects with symbol number -2. HTH

**Comment by dl3sdo** [comment]
You are right, besides the option to directly draw "layout vector objects" there are menu items to "Convert into [Graphic|Image|Layout] Object".

**Comment by dl3sdo** [comment]
Libor, thank you for your additions to import graphic objects. I created line and area objects, duplicated them and changed the duplicates to graphical objects. Your commits now allow to import these objects correctly.

I noticed that in Parameter String 1024 there is a flag that controls the visibility of these graphical objects.
I added a commit that imports this flag and applies it later on to the imported objects (I prefer thus to first import the flag and then import the objects, instead of changing the visibility afterwards):
https://github.com/dl3sdo/mapper/commit/a6ac39925015a564c68d3760832598fccc633abc

Please find attached my test files (also including layout vector objects) with enabled and disabled graphic object visibility.
[LayoutGraphicalObjects.zip](https://github.com/OpenOrienteering/mapper/files/8405194/LayoutGraphicalObjects.zip)

I intend to have a look on importing layout objects as well.

**Comment by dl3sdo** [comment]
I added the functionality of importing layout objects by re-using your code, Libor:
https://github.com/dl3sdo/mapper/commit/3cd57146c0d694f9bcd434864b00799b787c6cb9
The import of text attributes is still missing, but the format is clear (tabulator separated string attached to the text string itself).
If import of both object types shall become part of Mapper, then it's possible to merge the import functionality.

**Comment by dl3sdo** [comment]
@lpechacek: first, Happy New Year!
I finished the import of text layout objects by considering text attributes.
Please review and test the implementation.

Here is another test file:
[LayoutGraphicalObjects_Duplicates4.zip](https://github.com/OpenOrienteering/mapper/files/10352069/LayoutGraphicalObjects_Duplicates4.zip)
